cmd/nvidia-dra-plugin/sharing.go

```diff
@@ -101,17 +101,32 @@ func (t *TimeSlicingManager) SetTimeSlice(devices *PreparedDevices, config *nasc
 		return fmt.Errorf("setting a TimeSlice duration on MIG devices is unsupported")
 	}
 
+	noSupportTimeSliceIDs := []string{}
+	for _, gpu := range devices.Gpu.Devices {
+		if !detectSupportTimeSliceByArch(gpu.architecture) {
+			klog.InfoS("the current card does not support setting time slices and will be ignored.", "arch", gpu.architecture, "uuid", gpu.uuid, "cudaComputeCapability", gpu.cudaComputeCapability)
+			noSupportTimeSliceIDs = append(noSupportTimeSliceIDs, gpu.uuid)
+			continue
+		}
+	}
+
+	supportTimeSliceIDs := difference(devices.UUIDs(), noSupportTimeSliceIDs)
+
+	if len(supportTimeSliceIDs) == 0 {
+		klog.InfoS("all card does not support setting time slices and will be ignored.", "uuids", noSupportTimeSliceIDs)
+	}
```

Comment:
We're already logging the unsupported cards above and can remove this log message. We also don't need to calculate the difference if we construct a slice of supported IDs.

Reply:
At first I thought if supportTimeSliceIDs is empty, just return directly. Later I found out that it was not necessary, thanks for reminding me